src/main/java/tuteez/logic/commands/FindCommand.java

```diff
@@ -25,29 +26,31 @@ public class FindCommand extends Command {
 
     private static final Logger logger = LogsCenter.getLogger(FindCommand.class);
 
-    private final NameContainsKeywordsPredicate predicate;
+    private final Predicate<Person> combinedPredicate;
 
     /**
      * Creates a FindCommand to find persons with the specified {@code NameContainsKeywordsPredicate}.
      */
-    public FindCommand(NameContainsKeywordsPredicate predicate) {
-        this.predicate = predicate;
-        logger.fine("Initializing FindCommand with predicate: " + predicate);
+    public FindCommand(Predicate<Person> combinedPredicate) {
```

Comment:
LGTM but maybe the error message for find can be better written. As of now when i type "find" without any input the message does not inform users about the other functionalities of find.

Reply:
Another issue is when i type "find john" it says "no persons listed" and the person list is cleared.

Reply:
I've resolved this in my latest commit, thanks for spotting that!